src/rpc/blockchain.cpp

```diff
@@ -1413,6 +1419,98 @@ UniValue reconsiderblock(const JSONRPCRequest& request)
     return NullUniValue;
 }
 
+UniValue requestblocks(const JSONRPCRequest& request)
```

Comment:
The use cases for the RPC `requestblocks` API:
You start your peer with `auto-download-blocks = false`, you will only sync the headers then. You can selectively download blocks and eventually pass them through the signal (== ZMQ), use cases: experiments, SPV, light-client backend, ideally if you have a full validated node within your network and you want to selectively load blocks from that node (you don't want to validate everything again).